examples/official-site/sitemap.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+  <url>
+    <loc>https://sql.datapage.app/</loc>
+    <lastmod>2024-10-10</lastmod>
```

Comment:
Is it ok to hardcode the date like this ? Won't the prevent re-crawling when we update the page ?

Reply:
if it remains hard coded for too long it is an issue, for this first try it is ok